status.py

```diff
@@ -177,22 +183,27 @@
                     "Remotely_On_Off_Status": {
                         "tags": [ TAG.Remotely, TAG.On, TAG.Off, TAG.Status ],
                     },
-                    "Standby_Glycool_Unit_On_Off_Status": {
-                        "tags": [ TAG.Standby, TAG.Glycool, TAG.Unit, TAG.On, TAG.Off, TAG.Status ],
-                    },
                     "Standby_Unit_On_Off_Status": {
                         "tags": [ TAG.Standby, TAG.Unit, TAG.On, TAG.Off, TAG.Status ],
+                        "subclasses": {
+                            "Standby_Glycool_Unit_On_Off_Status": {
+                                "tags": [ TAG.Standby, TAG.Glycool, TAG.Unit, TAG.On, TAG.Off, TAG.Status ],
+                            },
+                        },
                     }
                 },
                 "tags": [ TAG.On, TAG.Off, TAG.Status ],
+                "parents": [BRICK.On_Status, BRICK.Off_Status],
```

Comment:
``On_Status`` is not defined.

Reply:
``On_Status`` is not defined but how did it pass the test?

Reply:
Oops! Forgot to remove the reference to `On_Status` and `Off_Status`; these passed tests because only the `rdfs:subClassOf` relationship gets added so it doesn't fall under the purview of the existing tests. I'll make sure we catch stuff like this in the future

Reply:
This is detected in a new `tests/test_class_structure.py`